Tweak/Setup/XENSetupBaseTableController.m

```diff
@@ -161,18 +161,14 @@ - (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger
 -(CGFloat)tableView:(UITableView *)tableView heightForRowAtIndexPath:(NSIndexPath *)indexPath {
     if (SCREEN_MAX_LENGTH < 667) {
         return 60.0;
-    } else if (SCREEN_MAX_LENGTH < 568) {
-        return 50.0;
-    } else {
+    }  else {
```

Comment:
Surely this removes handling for devices of display height < 568? The if/else-if here was just incorrectly ordered.

Reply:
Right now, on the current version of Xen-HTML, the return of 50 would never have happened no matter the device. Because that never would have happened previously, I would think that no **new** handling issues would occur due to this change; there would only be a problem if the issue was there before and has gone unnoticed until now.

Reply:
True... I've revised the way this class handles rotation just now for #128 , which fixes the issue you've brought up. As a result, I'll be closing this merge request; thank you for bringing that to my attention!